src/web-animations-next-player.js

```diff
@@ -30,6 +30,38 @@
 
   // TODO: add a source getter/setter
   scope.Player.prototype = {
+    _updateChildTiming: function(childPlayer, offset) {
+      if (childPlayer.startTime != this.startTime + offset) {
+        if (this.startTime === null) {
+          childPlayer.currentTime = this.source.player.currentTime - offset;
+          childPlayer._startTime = null;
+        } else {
+          childPlayer.startTime = this.startTime + offset;
+        }
+      }
+      // FIXME: What is this for?
```

Comment:
> Gooood question. What happens if we remove it?

Nothing, but I don't know if it is covered by unit tests.

I tested just with a little groups demo, seeking it to the end and then reversing it. That *looks* fine. I don't know if its state is correct w.r.t. to the model throughout.

It seems that if this code is useful for something it should be if (this.playbackRate < 0) not if (this.playbackRate == -1).

I'm guessing what it's meant to do is set the child player currentTime to -1 when the group is playing in reverse and the child is "done" (past its beginning). I don't know why we would want to do that. I also think that if it's a thing that needs doing it should happen in forEachChild, or a ticker, or both.